Makefile

```diff
@@ -48,23 +60,23 @@ up: ## Run the broker service with the brokerpak configured. The broker listens
 down: .env.secrets ## Bring the cloud-service-broker service down
 	@-docker stop csb-service
 
-# Normally we would run 
-# $(CSB) client run-examples --filename examples.json
-# ...to test the brokerpak. However, some of our examples need to run nested.
-# So, we'll run them manually with eden via "demo-up" and "demo-down" targets.
-test: examples.json demo-up demo-run demo-down ## Execute the brokerpak examples against the running broker
+# Normally we would just run `$(CSB) client run-examples` to test the brokerpak.
+# However, some of our tests need to run between bind and unbind. So, we'll
+# provision+bind and unbind+deprovision manually with eden via "demo-up" and
+# "demo-down" targets.
```

Comment:
This would also be good to document in the README. Sounds like it would be useful to get the thing up (provisioned and/or bound) as a setup for development.

Reply:
It's not very useful, though, because you can only mess around inside the provisioned EKS; you can't do anything with the Terraform code or state associated with it.